components/dada-ir/src/code/syntax.rs

```diff
@@ -161,6 +161,9 @@ pub enum ExprData {
     /// `a := b`
     Assign(Expr, Expr),
 
+    /// return
+    Return(Expr),
```

Comment:
We do want to support `return` by itself. The syntax tree would have `Option<Expr>`, I imagine, but the validator would substitute a `()` expression.